### ISSUE TITLE
Image data

### ISSUE THREAD

**Issue by mahdoosh1** [directmusic/wav2png#1]
when you convert a png file to a wave file you cant get image data back by wave file
so wav2png is lossy

**Comment by directmusic** [comment]
I do not believe I claim it is lossless anywhere. I assume you're simply dragging in an image with an alpha channel as images without do appear to work on my end. Could you provide an example of this occurring?

**Comment by NapalmCandy** [comment]
It works perfectly for me

**Comment by mahdoosh1** [comment]
i mean when you upload an image with rgb format it turns into just rg format and blue channel gets deleted

and image dimension is getting changed from original size to 3000 × any number

**Comment by mahdoosh1** [comment]
it was so good if every channel had its own unique sound wave

**Comment by mahdoosh1** [comment]
sorry my hand just clicked on close :/
